static/specification.html

```diff
@@ -410,11 +410,10 @@ <h2>
           <li>[=Queue an element task=] on the [=networking task source=] given
           |element| and the following steps:
             <ol>
-              <li>If |success| is true, [=fire an event=] named "load" at
-              |element|, which bubbles.
```

Comment:
@marcoscaceres 
FYI: 
![image](https://user-images.githubusercontent.com/525211/145168409-43628067-0164-43e6-a9ea-68aaa8c7345b.png)
Url out of date: 
https://webmonetization-preview.netlify.app/#link-type-monetization

Reply:
yeah, I need to set up a solution to generate the preview... bugging @AlexLakatos every time I make a change is not going to scale 😝

Reply:
Maybe can use https://wicg.github.io/webmonetization for spec preview and https://webmonetization.org for all the things?

Reply:
yeah, something like that would work...

Reply:
In the meantime, I've re-deployed, so now the preview is up to date.